<?xml version="1.0" ?>
<launch>        
    <!-- Controllers -->
    <node pkg="mh5_controller" type="mh5_controller.py" name="mh5_controller">
        <param name="config_file" value="$(find mh5_controller)/config/mh5.yml" />
	    <param name="rate" value="50" />
    </node>
    <!-- UI -->
    <!-- Warning: do not remove outpput="screen". It is needed so that the 
         UI is shown on the TTY -->
    <node pkg="mh5_ui" type="mh5_ui.py" name="ui" output="screen">
        <param name="rate" value="10" />
    </node>
</launch>
